plugins/tiddlywiki/prosemirror/slash-menu/slash-menu.js

```diff
@@ -211,41 +243,36 @@ function createSlashMenuPlugin(menuElements, options) {
 			handleDOMEvents: {
 				compositionstart: function(view, event) {
 					var state = SlashMenuKey.getState(view.state);
-					if (state && state.open) {
-						// Store that we're in composition mode for menu filtering
+					if(state && state.open) {
 						this._slashMenuComposing = true;
 					}
 					return false;
 				},
 				compositionupdate: function(view, event) {
 					var state = SlashMenuKey.getState(view.state);
-					if (state && state.open && this._slashMenuComposing) {
-						// Update filter with composition text
+					if(state && state.open && this._slashMenuComposing) {
 						var data = event.data;
-						if (data && data !== '、' && data !== '/') {
+						if(data && data !== "、" && data !== "/") {
 							dispatchWithMeta(view, {
 								type: SlashMetaTypes.inputChange,
 								filter: data
 							});
 						}
-						return true; // Prevent the composition from being inserted
```

Comment:
Why are the comments removed. If they are needed to better understand the code IMO they should stay.